website/content/en/docs/add-ons/storage/efs/guide.md

```diff
@@ -37,9 +39,18 @@ export CLAIM_NAME=<efs-claim>
 
 ## 2.0 Set up EFS
 
+#### Setup for Kustomize deployments
```

Comment:
Manifest instead of Kustomize? since it includes both kustomize and helm

same for other guides

Reply:
done